### ISSUE TITLE
Enums with one value cause failure

### ISSUE THREAD

**Issue by jdhornsby** [mprivat/xgql#3]
An enum with a single value causes a js error:

```
enum X {
    A
}
```

```
xgql merge --fill --style appsync test.graphql
(node:77003) UnhandledPromiseRejectionWarning: TypeError: enumTypeDefinition.values is not iterable
    at printEnumTypeDefinition (/Users/jdhornsby/.nvm/versions/node/v12.14.1/lib/node_modules/xgql/json-to-sdl.js:286:40)
    at toSDL (/Users/jdhornsby/.nvm/versions/node/v12.14.1/lib/node_modules/xgql/json-to-sdl.js:23:9)
    at run (/Users/jdhornsby/.nvm/versions/node/v12.14.1/lib/node_modules/xgql/xgql-merge.js:146:18)
(node:77003) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:77003) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by mprivat** [comment]
oh weird. I'll take a look.

**Comment by mprivat** [comment]
Fixed in 1.9.1
